integreat_cms/cms/templates/_base.html

```diff
@@ -29,7 +29,7 @@
             {% trans "Documentation (Wiki)" %}
         </a>
         <div id="region-selector"
-             class="relative flex flex-col justify-center px-2 cursor-pointer text-gray-800 hover:bg-gray-200 border-r border-gray-400">
+             class="hidden relative lg:flex flex-col justify-center px-2 cursor-pointer text-gray-800 hover:bg-gray-200 border-r border-gray-400">
```

Comment:
Here as well - we need at least some kind of region selector, otherwise it's not possible to navigate back to the network area from within a region.
(This is not only relevant for developers, but also for e.g. the service team members, who maybe also want to check a few things on the go)...

Reply:
I understand :)